src/cdf/helper.ts

```diff
@@ -3,15 +3,16 @@ import _ from 'lodash';
 export const filterLabels = (labels) =>
   !_.isEmpty(_.get(labels, 'containsAny')) && {
     labels: {
-      containsAny: labels.containsAny.map(({ value }) => ({ externalId: value })),
+      containsAny: _.map(labels.containsAny, ({ value }) => ({ externalId: value })),
     },
   };
-export const filterExternalId = (sourceExternalIds) =>
+
+export const filterExternalId = (sourceExternalIds, targetTypes: boolean) =>
   !_.isEmpty(sourceExternalIds) && {
-    targetTypes: ['timeSeries'],
+    targetTypes: targetTypes ? ['timeSeries'] : undefined,
```

Comment:
The variable name `targetTypes` seems vague considering that its a `boolean`, should be better off if renamed to `isTypeTimeseries` or `isTimeseries`